res/themes/dark/css/_dark.scss

```diff
@@ -286,6 +286,10 @@ $composer-shadow-color: rgba(0, 0, 0, 0.28);
     pre, code {
         filter: invert(1);
     }
+    a code{
+        color:#238CF5;
+
+    }
```

Comment:
This doesn't match our code-style, missing space after `:`, spurious newline.

Also the colour should be a reference to a variable and not a hardcoded hex value

Reply:
I've added the space and remove the new line. Although a new variable might be needed as `$input-focused-border-color` which doesn't seem suitable for text color